architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -506,3 +512,94 @@ link:../../install_config/install/prerequisites.html#host-preparation[Host
 Preparation] for information.
 ====
 endif::[]
+
```

Comment:
@bfallonf Lines 515-605 should be removed. They're getting worked on via https://github.com/openshift/openshift-docs/pull/1890. Hopefully I'll get 1890 merged soon so this will stop being a problem, sorry.